src/coreclr/src/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -253,9 +253,6 @@
     <Compile Include="$(BclSourcesRoot)\Internal\Runtime\InteropServices\ComActivator.cs" />
     <Compile Include="$(BclSourcesRoot)\Internal\Runtime\InteropServices\InMemoryAssemblyLoader.cs" />
   </ItemGroup>
-  <ItemGroup Condition="'$(FeatureUtf8String)' == 'true'">
```

Comment:
There is also a block in `System.Private.CoreLib.Shared.projitems` that should be deleted

Reply:
Thanks for pointing that out, removed in 3rd commit.